.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Please explain why you need to add it.

Reply:
i opened live server it is that change i think